MISRA.md

```diff
@@ -1,24 +1,16 @@
-
 # MISRA Compliance
 
 The jobs library files conform to the [MISRA C:2012](https://www.misra.org.uk)
 guidelines, with some noted exceptions. Compliance is checked with Coverity static analysis.
-Deviations from the MISRA standard are listed below:
-
-### Ignored by [Coverity Configuration](tools/coverity/misra.config)
-| Deviation | Category | Justification |
-| :-: | :-: | :-: |
-| Directive 4.9 | Advisory | Allow inclusion of function like macros. |
-| Rule 3.1 | Required | Allow nested comments. C++ style `//` comments are used in example code within Doxygen documentation blocks. |
-| Rule 20.12 | Required | Allow use of `assert()`, which uses a parameter in both expanded and raw forms. |
+The specific deviations, suppressed inline, are listed below.
 
-### Flagged by Coverity
-| Deviation | Category | Justification |
-| :-: | :-: | :-: |
-| Rule 2.5 | Advisory | A macro is not used by the library; however, it exists to be used by an application. |
-| Rule 8.7 | Advisory | API functions are not used by the library; however, they must be externally visible in order to be used by an application. |
+Additionally, [MISRA configuration file](https://github.com/aws/Jobs-for-AWS-IoT-embedded-sdk/blob/main/tools/coverity/misra.config) contains the project wide deviations.
 
 ### Suppressed with Coverity Comments
-| Deviation | Category | Justification |
-| :-: | :-: | :-: |
-| Rule 10.1 | Required | A variable of an enum type is used to iterate over contiguous values. |
+To find the violation references in the source files run grep on the source code
+with ( Assuming rule 11.4 violation; with justification in point 2 ):
+```
+grep 'MISRA Ref 11.1.4' . -rI
```

Comment:
Copied the grep from FreeRTOS+TCP MISRA changes as seen [here](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/blob/main/MISRA.md), as I was told to make the formatting of these files/MISRA changes the same across the various repos.
Since this repo currently doesn't have any suppressed violations I see nothing wrong with removing the grep and just stating that in the file for now, I had left it there to be consistent was all.
I imagine the general idea for the inclusion of the grep instead of the table is to handle line changes when commits happen. For example corePKCS11 has upwards of 50-60 inline suppressions, so adding a table to that would require a bit of overhead to ensure it stays accurate.